src/CDDL-1.0.xml

```diff
@@ -0,0 +1,397 @@
+<spdx name="Common Development and Distribution License 1.0" identifier="CDDL-1.0" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/cddl1</url>
+  </urls>
+  <notes>This license was released: 24 January 2004.</notes>
+  <license>
+    <title>
+      <p>COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) 
+        <br/>Version 1.0 
+      </p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions.</p>
+        </li>
+        <li>
+          <b>1.1.</b>
+          <p>&#x201C;Contributor&#x201D; means each individual or entity that creates or contributes to the
```

Comment:
This (http://stackoverflow.com/questions/1091945/what-characters-do-i-need-to-escape-in-xml-documents) provides a list of XML characters that need to be escaped, and the left and right quote are not amongst them, so in that case it ought to be valid to replace these with their proper UTF-8 characters.